profile/README.md

```diff
@@ -0,0 +1,30 @@
+<p align="center">
+    <img width="400px" src="https://raw.githubusercontent.com/FluxML/Flux.jl/master/docs/src/assets/logo.png"/>
+</p>
+
+# The Elegant Machine Learning Stack
+
+Flux is a 100% pure-Julia stack and provides lightweight abstractions on top of Julia's native GPU and AD support. It makes the easy things easy while remaining fully hackable.
+
+## Features
+
+Flux has features that sets it apart among ML systems
+
+### Compiled Eager Code
+Flux provides a single, intuitive way to define models, just like mathematical notation. Julia transparently [compiles your code](https://julialang.org/blog/2018/12/ml-language-compiler), optimising and fusing kernels for the GPU, for the best performance.
```

Comment:
```suggestion
Flux provides a single, intuitive way to define models, just like mathematical notation. Julia transparently [compiles your code](https://julialang.org/blog/2018/12/ml-language-compiler), optimizing kernels for the GPU, for the best performance.
```
There's been some confusion around where/when fusion happens before, so let's just avoid opening that can of worms here.